css/styles.css

```diff
@@ -0,0 +1,157 @@
+body {
+  font-family:
+    Roboto,
+    sans-serif;
+}
+
+.nav {
+  background-color: #80272c;
+}
+
+.nav-items-container {
+  height: 85px;
+  justify-content: space-evenly;
+  align-items: center;
+  color: #fff;
+}
+
+.nav-items {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav-items div {
+  margin-right: 20px;
+}
+
+.home {
+  font-size: 1.2rem;
+}
+
+#name {
+  padding: 6px 12px;
+  border-radius: 4px;
+  border: none;
+  font-size: 16px;
+  width: 200px;
+}
+
+#submit-btn {
+  padding: 6px 12px;
+  border-radius: 4px;
+  border: none;
+  font-size: 16px;
+  margin-left: 8px;
+}
+
+.main-characters {
+  text-align: center;
+}
+
+.char-tile-container {
+  display: flex;
+
+  /* flex-wrap: wrap; */
+  justify-content: center;
+}
+
+.character-tile {
+  flex-basis: calc(100% / 3);
+  text-align: center;
+  box-shadow: 0 3px 10px rgb(0 0 0 / 20%);
+  margin: 5px;
+}
+
+.character-name {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 50px;
+}
```

Comment:
For small stuff like font sizes and spacing, use `rem` over `px`: https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/effective-css